pkg/backend/local/progress.go

```diff
@@ -670,6 +670,10 @@ func (display *ProgressDisplay) processEndSteps() {
 
 				wroteResourceHeader := false
 				for _, v := range payloads {
+					if v.IsStatus {
```

Comment:
Does this work for the non-interactive view too?

Reply:
Yes.

Reply:
<img width="956" alt="screen shot 2018-08-30 at 4 17 30 pm" src="https://user-images.githubusercontent.com/1409156/44884431-3a589a80-ac70-11e8-8667-bfed19712f24.png">